### PR TITLE
remove abandoned connectors from docs

### LINKED ISSUE #1001: Disable abandoned connectors
Decision has been made to disable:
- sawtooth
- iroha
- burrow

Code:
- [x] Remove code for connectors [sawtooth/iroha/burrow]
- [x] Remove from publishing script
- [x] Remove from binding capability
- [x] Remove from CI
- [x] Add to changelog

Docs:
- [x] Remove front page logos
- [x] Remove related docs entries

Sample benchmarks:
- [x] Remove all

### PR DESCRIPTION
**Pull request by nklincoln**
closes #1001 

Signed-off-by: nkl199@yahoo.co.uk <nkl199@yahoo.co.uk>

### CODE REVIEW

File: docs/vNext/architecture_CH.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ order: 6
 
 ### Adaptation Layer(适配层)
 
-适配层用于将现有的区块链系统集成到Caliper框架中。每个适配器使用相应的区块链SDK或RESTful API实现'Caliper Blockchain NBI'。目前支持Hyperledger Fabric1.0-1.4、Sawtooth、Iroha、和burrow。Caliper后续将实现对以太坊和其他区块链系统的支持。
+适配层用于将现有的区块链系统集成到Caliper框架中。每个适配器使用相应的区块链SDK或RESTful API实现'Caliper Blockchain NBI'。目前支持Hyperledger Fabric1.0-1.4、Caliper后续将实现对以太坊和其他区块链系统的支持。
```
Comment (aklenik) [inline docs/vNext/architecture_CH.md:15]: I think it's safe to deprecate this page, the content is way outdated.

File: index.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
 <div class="top-jumbo">
   <div class="left-jumbo">
     <h1>Hyperledger Caliper</h1>
-    <h2>Blockchain performance benchmarking for Hyperledger Besu, Burrow, Fabric, Iroha & Sawtooth, and non Hyperledger Ethereum and FISCO BCOS networks</h2>
+    <h2>Blockchain performance benchmarking for Hyperledger Besu, and Fabric, and non Hyperledger Ethereum and FISCO BCOS networks</h2>
```
Comment (aklenik) [inline index.html:8]: I wouldn't split the enumeration based on HL. I'd use the full name everywhere, or just HL, like: HL Besu, HL Fabric, Ethereum and FISCO BCOS

File: index.html [hunk 3]
```diff
@@ -88,10 +88,6 @@ <h2>Jump into using Caliper with the other projects</h2>
         <strong>Hyperledger Besu</strong>
         <img src="https://wiki.hyperledger.org/download/thumbnails/16326961/Hyperledger_Besu_color.png?version=1&modificationDate=1567198581000&api=v2" alt="Burrow">
       </a>
-      <a class="project" href="{{ site.baseurl }}/{{ site.caliper.current_release }}/burrow-config">
-        <strong>Hyperledger Burrow</strong>
-        <img src="https://www.hyperledger.org/wp-content/uploads/2018/06/Hyperledger_Composer_Logo_White.png" alt="Burrow">
-      </a>
       <a class="project" href="{{ site.baseurl }}/{{ site.caliper.current_release }}/ethereum-config">
         <strong>Ethereum</strong>
```
Comment (aklenik) [inline index.html:92]: The text is next to the image, looks a bit weird.

**Review by aklenik — CHANGES_REQUESTED** [review]
Just some minor, debatable comments

File: index.html (hunk 3 above)
Comment (nklincoln) [reply index.html:92]: hmmm, that one is a sizing issue and likely to be a style thing with flex rather than a text placement issue

Comment (nklincoln) [reply index.html:92]: addition of `display: flex` and `flex-direction: column` 👍

**Review by aklenik — APPROVED** [review]
👍

**Pull request merged**